### ISSUE TITLE
Questions about training

### ISSUE THREAD

**Issue by LX0912R** [wang-jh18-SVM/Griffin#12]
Hello, thank you very much for your contribution to the community and for sharing this code.

Following the documentation, I configured the dataset and obtained the same evaluation results as reported in the paper using the provided checkpoint. However, I encountered an issue when reproducing the training process, as follows.

Based on my hardware setup, I modified the config file by setting `num_gpus = 1` and `batch_size = 8`, while keeping all other parameters unchanged. I then executed the command: `python tools/train.py --config projects/configs_griffin_50scenes_25m/vehicle-side/tiny_track_r50_stream_bs8_48epoch_3cls.py --work-dir work_dirs --deterministic True`.

However, during training, the AMOTA in the validation phase remains 0. Could you provide any suggestions regarding this issue?

<img width="1843" height="984" alt="Image" src="https://github.com/user-attachments/assets/03d2c95d-4f79-415c-992c-f90fcad137e0" />

**Comment by wang-jh18-SVM** [comment]
Hello, and thank you for the detailed feedback.

Since you confirmed that the provided checkpoint produces the correct evaluation results, we can rule out issues with the data conversion or the core codebase; your environment and data preparation seem correct.

Regarding the training issue, here are a few thoughts:
1.  Convergence Speed: Tracking metrics often rise slower than detection metrics. However, having 0 AMOTA after 16,000+ steps is indeed abnormal. Even with a slow start, we should usually see some non-zero metrics by that point.
2.  Batch Size & Learning Rate: Regarding your modification from a batch size of $2 \times 4$ GPUs to $8 \times 1$ GPU: since the total effective batch size remains constant (8), you theoretically do not need to adjust the learning rate. The optimization dynamics should be very similar, so this is likely not the root cause.

Given this, the cause is not immediately obvious. To help me diagnose this further, could you please share the full modified config file and the training log?

Note on logging: I recommend running the training using `tools/dist_train.sh` to ensure the detailed logs are captured correctly (it uses `tee` to save the output). Since you are using a single GPU, you can simply set the second parameter to 1:
```bash
bash tools/dist_train.sh <YOUR_CONFIG_PATH> 1
```

I will try to reproduce this error on my server once I have these details.

**Comment by LX0912R** [comment]
First of all, thank you very much for your prompt response. Following your suggestion, I reproduced the issue again, as follows:

1. Commented out the line `--launcher python ${@:3}` in `dist_train.sh`, otherwise it would trigger `distributed=True` in `train.py`.

2. `bash tools/dist_train.sh projects/configs_griffin_50scenes_25m/vehicle-side/tiny_track_r50_stream_bs8_48epoch_3cls.py 1`

The complete log and the modified config file are here [logs & config.zip](https://github.com/user-attachments/files/23668519/logs.config.zip)

Once again, I extend my sincere thanks.

**Comment by zzyadad** [comment]
Have you solved your problem? I have encountered the same issue.

**Comment by wang-jh18-SVM** [comment]
If you are looking for a model with better performance, I suggest referring to [SparseCoop](https://github.com/wang-jh18-SVM/SparseCoop)
